doc/docs.md

```diff
@@ -802,7 +814,7 @@ instead of
 new_button(ButtonConfig{text:'Click me', width:100})
 ```
 
-This only works with functions that have a single struct argument.
+This only works for functions that have a struct for the last argument.
```

Comment:
no, not last, there needs to be just one struct arg

Reply:
Preceding arguments do work:
https://github.com/vlang/v/pull/5726/files#diff-be137ec37ed574953500274b6e369f46R253

Is this a compiler bug? I think it's useful behaviour.

Reply:
Hm indeed it does. Strange. I'll need to think about it.